Week4/ReadingList/app/src/main/java/com/florencenjeri/readinglist/ui/BooksDetailsFragment.kt

```diff
@@ -75,4 +80,13 @@ class BooksDetailsFragment : Fragment() {
         Log.d("Books", book.toString())
         booksViewModel.deleteBook(book)
     }
+
+    //Animation Container Transform
+    private fun buildContainerTransform() =
+        MaterialContainerTransform().apply {
+            drawingViewId = R.id.nav_host_fragment
+            duration = 30000000
```

Comment:
Try to pull these out into constants! Also, this seems like a really big number!